CHANGES/1200.feature.rst

```diff
@@ -0,0 +1,7 @@
+Skipped reallocation of internal htkeys structure on inserting new items if the
```

Comment:
Can't we just use inline highlighting for things that clearly reference source code identifiers? This way, we wouldn't have to keep adding special cases to the known words list. This isn't a word, anyway. So let's not pretend like it is.
```suggestion
Skipped reallocation of internal ``htkeys_t`` structure on inserting new items if the
```

Reply:
ok, thanks!